prow/plugins/plugins.go

```diff
@@ -483,11 +483,12 @@ func (pa *PluginAgent) Load(path string) error {
 	if err := yaml.Unmarshal(b, np); err != nil {
 		return err
 	}
-
 	if len(np.Plugins) == 0 {
 		logrus.Warn("no plugins specified-- check syntax?")
 	}
-
+	if err := ensureConfig(np); err != nil {
+		return err
+	}
```

Comment:
i think its fine to call here, but it still necessary to do before `np.setDefaults`